app/backend/src/couchers/servicers/requests.py

```diff
@@ -136,7 +136,7 @@ def CreateHostRequest(self, request, context):
 
             return requests_pb2.CreateHostRequestRes(host_request_id=host_request.conversation_id)
 
-    def GetHostRequest(self, request, context):
+    def GetHostRequest(self, request, context, pb=True):
```

Comment:
Don't do this, please. It's confusing, this does two very different things now which is not good conceptually.

Just make a new helper function to grab the host request if you want to